### PR TITLE
add spacing mdx

### PR DESCRIPTION
**Pull request by jonborchardt**

### CODE REVIEW

File: demo/src/pages/Breakpoints/demo/default.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import styled from 'styled-components';
 import { belowOrEqualTo } from '@allenai/varnish/theme/breakpoints';
 
 const BreakpointDiv = styled.div`
-    width: 100%;
+    width: 90%;
```
Comment (codeviking) [inline demo/src/pages/Breakpoints/demo/default.mdx:13]: Is this actually `%` (in that it's of it's parent), or is this a place where we might want to use `vw` units?

File: demo/src/pages/Spacing/SpacingValues.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        return (
+            <>
+                <Namespace>
+                    <BodyMicro>theme.spacing.</BodyMicro>
+                </Namespace>
+                <SpacingGrid>
+                    {Object.keys(spacing).map((sKey) => {
+                        const spac = spacing[sKey];
+                        return <SpacingRow spacing={spac} key={sKey} label={sKey} />;
+                    })}
```
Comment (codeviking) [inline demo/src/pages/Spacing/SpacingValues.tsx:18]: I think `Object.entires` could work here too.

File: demo/src/pages/Breakpoints/demo/default.mdx (hunk 1 above)
Comment (jonborchardt) [reply demo/src/pages/Breakpoints/demo/default.mdx:13]: yes of parent....

File: demo/src/pages/Spacing/SpacingValues.tsx (hunk 2 above)
Comment (jonborchardt) [reply demo/src/pages/Spacing/SpacingValues.tsx:18]: will do

File: demo/src/pages/Spacing/page.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+iconSrc: ./spacing-14px.svg
+---
+
+import defaultDemo from './demo/default.mdx'; // crazy, we must import for the require to work
+const mdxs = require.context('./demo', false, /\.(mdx)$/);
+const examples = mdxs.keys().map(mdxs);
+
+## Spacing
+
+Standard AI2 spacing units.
```
Comment (codeviking) [inline demo/src/pages/Spacing/page.mdx:14]:
```suggestion
Standard AI2 spacing units. Use these values to create space between and/or inside of elements. If you're setting `margin` or `padding`, you should probably be using one of these values. We've selected values that when used together create a consistent visual rhythm, making the resulting site look better and easier to scan.
```

**Review by codeviking — APPROVED** [review]

**Pull request merged**